CharacterCreation/Class-Specific Documentation/Medic Proceedures.txt

```diff
@@ -250,7 +251,8 @@ Duration: 	almost instantaneous
 Range:		Payload (use in combination with other powers)
 Effect:		A biologic target afflicted with this poison takes 180 damage that 
 ignores shields, silences the target and prevents them from casting nanite 
-spells for 10 turns if they survive.
+spells for 10 turns if they survive. (Also prevents engineering processes and 
```

Comment:
Should you change 

>nanite spells

to "Nanite Abilities"? Since we are moving away from the DnD 'spells' aesthetic